cmd/demoserver/kvdatamake.go

```diff
@@ -281,6 +284,7 @@ func (s *DataRWServer) ReadLines(req *pb.ReadLinesRequest, stream pb.DataRWServi
 		case <-s.ctx.Done():
 			return nil
 		default:
+			time.Sleep(time.Millisecond * 5)
```

Comment:
why sleep here

Reply:
Slower the read speed to mitigate the pressure of disk.